tests/browser/test/basic_playground_test.js

```diff
@@ -182,7 +182,7 @@ suite('Disabling', function () {
         110,
         110
       );
-      await connect(browser, child, 'PREVIOUS', parent, 'IF0');
```

Comment:
Huh, I guess this was just getting it close enough that it was working? lol

Reply:
Yup.